src/CMakeLists.txt

```diff
@@ -587,15 +585,22 @@ if(${WITH_RADOSGW})
     cls_statelog_client
     cls_rgw_client
     fcgi
-    ${CURL_LIBRARIES}
     ${EXPAT_LIBRARIES}
   )
 endif(${WITH_RADOSGW})
-if(${WITH_RBD})
+if(WITH_RBD)
   set(DENCODER_EXTRALIBS
       ${DENCODER_EXTRALIBS}
       rbd_replay_types)
-endif(${WITH_RBD})
+  if(LINUX)
+    list(APPEND dencoder_srcs
```

Comment:
i understand that the freebsd build does not support krbd, but could you explain this in the commit message also?